react/src/components/HeaderMenu/HeaderMenuContainer.tsx

```diff
@@ -23,40 +23,50 @@ const HeaderMenuContainer = () => {
   const menuItems: MenuItemsGroupData[] = useMemo(
     () => [
       {
-        text: "Boxes",
+        text: "Classic View",
         links: [
-          // { link: "#", name: "Print Labels" },
-          { link: `/bases/${baseId}/boxes`, name: "Manage Boxes" },
-            // TODO: uncomment this once we have finished/tested the Create Box feature sufficiently
-          // { link: `/bases/${baseId}/boxes/create`, name: "Create new Box" },
-          // { link: "#", name: "Stock Overview" },
+          {
+            link: `${process.env.REACT_APP_OLD_APP_BASE_URL}/mobile.php`,
```

Comment:
I suppose this env variable needs to be set for the deployed environments. You can add a line like 

```         REACT_APP_OLD_APP_BASE_URL=${OLD_APP_BASE_URL}
```
in line 65 in .circleci/config.yml. I can populate these values in the CI configuration.

Reply:
@pylipp 
@johnsmol is not used to the CircleCI bit. I would ask you to add the line to config and populate the values in CircleCI.

Reply:
:+1: I set `https://staging.boxtribute.org` for `OLD_APP_BASE_URL` in the CircleCI staging context, `https://app.boxtribute.org` for production, and `https://demo.boxtribute.org` for demo context.